### ISSUE TITLE
Contextual details of food production and harvesting

### ISSUE THREAD

**Issue by Public-Health-Bioinformatics** [FoodOntology/foodon#18]
Matthew's point:

> 1. We need a way to describe how/where/when the food was
>    grown/produced/procured.
>    - It is not enough to say it is a freshwater fish or marine
>      fish--salmon is an example of an organism that requires a more robust
>      characterization that coincides with season and life-stage
>      --this will make a difference as to its composition..
>    - Another example is Coturnism, an illness known since ancient times,
>      that results in rhabdomyolysis (muscle cell breakdown) from eating quail
>      that have fed on poisonous plants. The quail are not noticeably different
>      by either taste or visual senses,  but Sinai hunters familiar with their
>      migration patterns generally know WHEN the quail are safe.

I've seen decent environmental descriptors for:
biome http://purl.obolibrary.org/obo/ENVO_00000428
terrain http://purl.obolibrary.org/obo/ENVO_00000000
season http://purl.obolibrary.org/obo/EO_0007027
life cycle: http://purl.obolibrary.org/obo/UBERON_0000105

A while back I created a [Food Schema](https://docs.google.com/spreadsheets/d/12cnHZZJc8Y-5QtbE3NGGlgD0zqy-eQPuMB-tMCMjCAA/edit?usp=sharing) spreadsheet to try to anticipate the contextual info as a list of descriptors or facets (with new ones needed like "Collection technology").

So first question is are people comfortable with a "food life cycle" that has similarities to the general "life cycle" categories above?  They would be anthropocentric distinctions, something like:
**Pre-collection food state** or "Pre-collection food environment (physical context)*\* and related qualities, followed by
**Collection food state (harvested/slaughtered)** or **Collection food process (harvest/slaughter)** and qualities thereof.

Roughly we need to express statements about a given food item's pre-collection environment that are indicators of other food qualities (that need to be defined, but would include food safety).  And we need statements about collection conditions that bear on food quality (also including food safety)?

**Comment by ddooley** [comment]
FoodOn now has  [pre-harvest food material](http://purl.obolibrary.org/obo/FOODON_00003402) and harvestable and harvested food material as siblings, to cover this roughly. I think it is up to pre-harvest agriculture/agronomy ontologies to further distinguish pre-harvested organism state.  We do however have a [degree of fruit maturity](http://purl.obolibrary.org/obo/FOODON_03530268) term and a degree of plant maturity term is under debate (i.e. how to incorporate with PATO "maturity").

**Comment by marieALaporte** [comment]
Shouldn't these pre-harvest food material and harvestable food material classes be defined classes? So when I collect a food, if I specify its level of maturity, it will be automatically classified under the correct class?
Also just reading the name 'pre-harvest food material', I first found it confusing in the sense that I was expecting that you were meaning food that is still on the plant or the animal or any other food source. Shouldn't the name mention immature instead of pre-harvest that is heavily connoted?

**Comment by ddooley** [comment]
I'd like to have them be defined classes, enabling inference.  Does AGRO have a way to do this now? I went to PATO yesterday to see what they had and found only [maturity](https://ontobee.org/ontology/PATO?iri=http://purl.obolibrary.org/obo/PATO_0000261) and "mature"; all of "maturity"'s children referenced animals, not plants, and there was no"immature".  

**ADDENDUM: Oddly, I only now see "immature" under PATO maturity. I must have been blind yesterday!**

So I was actually ducking the complexity of defining "immature" from PO plant context, and realized one other issue - harvesting of fruits often happens when they are immature, and ripening occurs later or during transport.  So that was the motivation to use harvestable, pre-harvest and harvested.  Pointing to what was actually done to the food rather than its stage of development.  

I tried weaving some processes in there, e.g. 

[food harvesting](https://ontobee.org/ontology/FOODON?iri=http://purl.obolibrary.org/obo/FOODON_00003203) subClassOf : [has_specified_input](https://ontobee.org/ontology/FOODON?iri=http://purl.obolibrary.org/obo/OBI_0000293) some [pre-harvest food material](https://ontobee.org/ontology/FOODON?iri=http://purl.obolibrary.org/obo/FOODON_00003402)

and 

[harvested food material](http://purl.obolibrary.org/obo/FOODON_00003374) subClassOf: [output of](https://ontobee.org/ontology/FOODON?iri=http://purl.obolibrary.org/obo/RO_0002353) some [food harvesting](https://ontobee.org/ontology/FOODON?iri=http://purl.obolibrary.org/obo/FOODON_00003203)

**Comment by ddooley** [comment]
Indeed, the intention is that we can describe an apple, and by saying it is pre-harvested food material, mean that it is attached to a tree still; and same with tomato; but also wheat in the ground, potatoes, peanuts in the ground.  Trying to cover food that may even be stored in the ground and not attached to its nurturing plant anymore.  Should I put more examples into "pre-harvest food material"?!   

And the purpose of "harvestable" is that an agriculturalist knows the food is ready for harvesting - whether it is immature (e.g. [climacteric fruit](https://extension.umd.edu/resource/ethylene-and-regulation-fruit-ripening#:~:text=Fruits%20are%20generally%20divided%20in,fruits%20cannot%20ripen%20after%20harvest.)) or mature.  Does that make sense as a category?

The other motivation for this is that sampling for nutritional content can happen at any one of those three states (or four if one counts pre-harvest and not harvestable).  That was coming from CDNO's need for describing biosamples.  Possibly this is needed for PTFI too?
